packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -271,6 +271,7 @@ class GroupedSingleSelectInput extends React.Component {
         <SelectOption key={3} value="Degraded" />
         <SelectOption key={4} value="Needs Maintenence" />
       </SelectGroup>,
+      <Divider/>,
```

Comment:
Would you use it by placing `<Divider/>` like this in the menu? Curious if this should align with the dropdown's `<DropdownSeparator>`. Also is there an example of how to include a divider between regular select items that are either in a single group, or in a single list with no groups?

Reply:
By align do you mean create a component `<SelectSeparator>`?  If so that is what I started with.  I thought about it more and decided against it.  It would just be a wrapper.  I think we kept `<SelectSeparator>` once the dropdown was refactored to use Divider in core in order to minimize breaking changes.  I was thinking in our next breaking change release we could update Dropdown and any other components that wrap the Divider.   I have no strong feeling either way.

I can added a a divider to the  "Checkbox input" example.

Reply:
OK that makes sense, thanks for the clarification. I wasn't sure if having a select specific component would allow us to do smarter detection of where the divider is placed as to which element the divider should render as. In the grouped and checkbox examples, it's fine for the divider to render as an `<hr>` (the default). But in a regular list where each item is an `<li>`, the divider would need to be an `<li>`, too. Do you think we should include an example of that to illustrate that the divider element needs to have `component="li"`?